content/spin/install.md

```diff
@@ -21,7 +22,7 @@ Spin runs on Linux (amd64 and arm64), macOS (Intel and Apple Silicon), and Windo
 
 {{ startTab "Linux"}}
 
-There are multiple ways to install Spin. The easiest is to use the installer script, hosted on this site:
+There are multiple ways to install Spin. The easiest is to use the installer script, hosted on this site. The script installs Spin along with [default templates and plugins](#installing-default-templates-and-plugins).
```

Comment:
I think if we remove this link, and then change the header of the template install not not use the word "Default" it feels less of a task I MUST do. And since most will use the install script (hypothesis), that extra section doesn't matter to most.

Reply:
Thanks Kate, looks great.